API/api/models.py

```
@@ -1,3 +1,93 @@
+
+    def __str__(self) -> str:
+        return self.name or f"Plage #{self.pk}"
+
+
+class Slot(models.Model):
+    plage = models.ForeignKey(Plage, on_delete=models.CASCADE, related_name="slots")
+    start_at = models.DateTimeField()
+    end_at = models.DateTimeField()
+
+    # Réservation (NULL = libre)
+    client_fname = models.CharField(max_length=120, blank=True, null=True)
```

Ruff (0.14.14)

[warning] Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)